### PR TITLE
Remove references to Run on Kubernetes in IntelliJ

### PR DESCRIPTION
**Pull request by kelsk**
As part of adding "On demand" watch mode to IntelliJ, the **Run on Kubernetes** run config has been removed.

The samples READMEs have been updated to remove any reference to Run on Kubernetes, including images.

Inconsistent formatting has been fixed ('single quotes' have been replaced with **bolding**).

### CODE REVIEW

File: dotnet/dotnet-guestbook/README.md [hunk 1]
```diff
@@ -62,8 +63,9 @@ For details on how to use this sample as a template in Cloud Code, read the docu
 
 
 #### Run the app on minikube
-1. Select 'Develop on Kubernetes' from the configuration drop down and click the run icon. Cloud Code runs your app in a local [minikube](ttps://minikube.sigs.k8s.io/docs/start/) cluster.  
-![image](./img/deploy-config.png)
+1. Select 'Develop on Kubernetes' from the configuration dropdown and click the run icon. Cloud Code runs your app in a local [minikube](ttps://minikube.sigs.k8s.io/docs/start/) cluster.  
+![image](./img/edit-configurations.png)
+
```
Comment (emmanuelbaah) [inline dotnet/dotnet-guestbook/README.md:68]: Missed a single quotes being replaced with bolding here

File: java/java-guestbook/README.md [hunk 2]
```diff
@@ -50,20 +62,22 @@ For details on how to use this sample as a template in Cloud Code, read the docu
...
-4. Click 'OK' to save your configuration. 
+4. Click **OK** to save your configuration. 
 
 
 #### Run the app on minikube
-1. Select 'Develop on Kubernetes' from the configuration drop down and click the run icon. Cloud Code runs your app in a local [minikube](ttps://minikube.sigs.k8s.io/docs/start/) cluster.  
-![image](./img/deploy-config.png)
+1. Select 'Develop on Kubernetes' from the configuration dropdown and click the run icon. Cloud Code runs your app in a local [minikube](ttps://minikube.sigs.k8s.io/docs/start/) cluster.  
+![image](./img/edit-configurations.png)
+
```
Comment (emmanuelbaah) [inline java/java-guestbook/README.md:80]: Another single quotes being replaced with bolding missed here

**Comment by emmanuelbaah** [comment]
Other than those 2 nits looks good!

File: dotnet/dotnet-guestbook/README.md (hunk 1 above)
Comment (kelsk) [reply dotnet/dotnet-guestbook/README.md]: Done

File: java/java-guestbook/README.md (hunk 2 above)
Comment (kelsk) [reply java/java-guestbook/README.md]: Thanks for catching these! Done

**Review by emmanuelbaah — APPROVED** [review]
lgtm!

**Pull request merged**